caiman/source_extraction/cnmf/cnmf.py

```diff
@@ -320,6 +320,8 @@ def fit_file(self, motion_correct=False, indices=[slice(None)]*2):
         Returns:
             cnmf object with the current estimates
         """
+        if indices is None:
```

Comment:
Any reason for removing this from the function definition?

Reply:
Yes,

Mutable arguments in function definitions can result in very surprising behavior. See [this](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument?noredirect=1&lq=1) and [this](http://www.omahapython.org/IdiomaticPython.html#default-parameter-values), for example.

Reply:
:+1: